test/test.js

```diff
@@ -289,6 +289,41 @@ test('fails if cron pattern is invalid', (t) => {
   );
 });
 
+test.serial('job created with cron string is using local timezone', (t) => {
+  t.plan(1);
+  const bree = new Bree({
+    root,
+    jobs: [{ name: 'basic', cron: '0 18 * * *' }]
+  });
+
+  const clock = FakeTimers.install({ now: Date.now() });
+  bree.start('basic');
+  bree.on('worker created', () => {
+    const now = new Date(clock.now);
+    t.is(now.getHours(), 18);
```

Comment:
let's change this to the timezone offset instead, so that we are testing explicitly what we want instead of an after effect. you should be able to use `now.getTimezoneOffset()`

Reply:
want to be testing the actual offset not just what the hours say.

Reply:
Ok, I added the offset check in the tests.